ProjectName/App.tsx

```diff
@@ -0,0 +1,58 @@
+// In App.js in a new project
+
+import * as React from 'react';
+
+import {NavigationContainer} from '@react-navigation/native';
```

Comment:
Do you have any separate task on board for the ES/TS linter? 
You can use this linter plugin to follow import rules https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/order.md